code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -610,7 +610,7 @@ Then we space some of our heat, and think about if we should stop conducting.
 	return ..()
 
 /turf/closed/consider_superconductivity(starting)
-	if(temperature < (starting?MINIMUM_TEMPERATURE_START_SUPERCONDUCTION:MINIMUM_TEMPERATURE_FOR_SUPERCONDUCTION))
+	if(temperature < (starting?MINIMUM_TEMPERATURE_START_SUPERCONDUCTION:MINIMUM_TEMPERATURE_FOR_SUPERCONDUCTION) || temperature < (heat_capacity * 0.95))
```

Comment:
I don't like this, but it will "work". See main post

Maybe you could override `super_conduct` and do this temperature check there instead? if it fails just radiate heat to space.

That way walls aren't perma heat stores, but they don't transfer to their neighbors before a threshold